settings.gradle

```diff
@@ -1,8 +1,15 @@
+pluginManagement {
```

Comment:
Why are you adding this into settings.gradle?

Reply:
I was messing with vanilla gradle at one point, gone now.